osidb/api_views.py

```diff
@@ -742,6 +743,28 @@ class FlawCVSSView(
     permission_classes = [IsAuthenticatedOrReadOnly]
     filterset_class = FlawCVSSFilter
 
+    def create(self, request: Request, *args: Any, **kwargs: Any) -> Response:
+        request.data.pop("issuer", None)
+        return super().create(request, *args, **kwargs)
+
+    def update(self, request: Request, *args: Any, **kwargs: Any) -> Response:
+        cvss: FlawCVSS = self.get_object()
+        if cvss.issuer == FlawCVSS.CVSSIssuer.REDHAT:
+            return super().update(request, *args, **kwargs)
+        return Response(FlawCVSSSerializer(cvss).data)
+
+    @extend_schema(
+        responses={
+            200: {},
+        },
+        parameters=[bz_api_key_param],
```

Comment:
I am thinking whether the `bz_api_key_param` is even needed anymore, all communication towards Bugzilla is performed via OSIDB Bugzilla service account with the exception of adding comments and filing trackers. We have this parameter set for a lot of endpoints and we should clean them up properly eventually.

Reply:
I added it just because the view class has a decorator that adds it to all methods, but when overriding the method it doesn't which led to backwards-incompatible changes in the schema.

Didn't know it was removed though, good to know!